Descriptors/EvaluationPeriodDescriptor.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<InterchangeDescriptors xmlns="http://ed-fi.org/3.3.1-b" xmlns:ann="http://ed-fi.org/annotation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://ed-fi.org/3.3.1-b ../Schemas/Bulk/EXTENSION-Interchange-Descriptors-Extension.xsd">
```

Comment:
For any of the quarterly values, could we match the fully spelled out version of these as we have in the Terms? I realize Period has the shortened CodeValue and ShortDescription so maybe just in the Description for consistency.